### ISSUE TITLE
Upgrade to Angular 13 and Ivy

### ISSUE THREAD

**Issue by wertzui** [ngstack/code-editor#818]
After upgrading my project, to Angular 13, `ng build` suggests, I should encourage you to support an Ivy distribution 😃 

```
Processing legacy "View Engine" libraries:
- @ngstack/code-editor [es2015/esm2015] (https://github.com/ngstack/code-editor.git)
Encourage the library authors to publish an Ivy distribution.
```

**Comment by DenysVuika** [comment]
thanks for the reminder, I will try to upgrade soon

**Comment by DenysVuika** [comment]
Updated and released as 4.0.0
